src/test/pybind/test_rados.py

```diff
@@ -598,6 +598,26 @@ def test_locator(self):
         eq(objects, [])
         self.ioctx.set_locator_key("")
 
+    def test_operate_aio_write_op(self):
+        lock = threading.Condition()
+        count = [0]
+        def cb(blah):
+            with lock:
+                count[0] += 1
+                lock.notify()
+            return 0
+        write_op = self.ioctx.create_write_op()
+        write_op.write(b'rzx')
+        comp = self.ioctx.operate_aio_write_op(write_op, "object", cb, cb)
+        comp.wait_for_complete()
+        comp.wait_for_safe()
```

Comment:
i don't think `wait_for_safe()` is necessary.

Reply:
@tchaikov Updated. 
Thanks a lot.